server/search/APIController.js

```diff
@@ -0,0 +1,137 @@
+const axios = require('axios');
+const API_KEY = process.env.API_KEY;
+module.exports = {
+
+  getHistory: (movieName) => {
+
+    const name = movieName.replace(' ', '%20');
+    const url = `https://api.themoviedb.org/3/search/movie?api_key=${API_KEY}&language=en-US&query=${name}&page=1&include_adult=false`;
+
+    return axios.get(url)
+      .then(({data}) => {
+
+        const {id, title, overview, poster_path, vote_average, vote_count} = data.results[0];
+        const imgUrl = `https://www.themoviedb.org/t/p/w1280${poster_path}`;
```

Comment:
We should not be using hi-res images for everything. Everything other than MediaDetail should be able to get away with `w200` or `w500`.